exercises/concept/simple-calculator/simple_calculator_test.rb

```diff
@@ -27,7 +27,7 @@ def test_no_number_second_operand_raises_exception
   end
 
   def test_raises_exception_for_non_valid_operations
-    assert_raises(SimpleCalculator::UnsupportedOperation) { SimpleCalculator.calculate(1, 2, '**') }
+    assert_raises(SimpleCalculator::UnsupportedOperation) { SimpleCalculator.calculate(1, 2, '*******') }
```

Comment:
This test should not be changed, and it is the test that says "Our library does not support this operation." 

However, instead of changed, I could see an argument for removing it, since yes, we could expand supported operations.

For the goals of this exercise, since we are not specifically aiming to support "ruby like" operators, and only generic, this test still allows for the exploration of allowing something like `'^'` to be added to the supported list of operators, while disallowing this specifically for the library (but not for Ruby itself, a different subject, really.)

Reply:
> This test should not be changed, and it is the test that says "Our library does not support this operation."

I don't follow this reasoning... the test title says that `the program should raise an exception for invalid operations` - not quite "our library does not support _this_ operation". This change more closely reflects the behaviour implied by the test's title, IMO. It'd be clearer to change the title of the test if the quoted statement is the intention, and perhaps add more assertions of non-standard/generic operations.

> For the goals of this exercise, since we are not specifically aiming to support "ruby like" operators, and only generic,

This is fair enough, so I won't say any more. Thanks for feedback and conversation.

Reply:
Yes, but the README and test are talking about our library, not Ruby specifically (We aren't testing Ruby, we are writing a program using Ruby).

For the change of the title of the test, that should be discussed in a way that makes sense for the tracks using the exercise, I think.  That would be in https://github.com/exercism/problem-specifications where the discussion and change would happen, hopefully, to benefit the other tracks as well.

And yes, of course, you are welcome to say more.  (And especially in the track exercise, during mentoring.)